.github/workflows/pr_test_cpu.yml

```diff
@@ -42,6 +42,18 @@ jobs:
       pytorch-dtype: 'float32,float64'
       coverage: true
       continue-on-error: true
+      pytest-extra: '--runslow'
+
+  test-slow:
```

Comment:
this isn't a necessary job, because the `coverage` already will run within the same parameters but with coverage report